src/widget/form/addfriendform.h

```diff
@@ -61,23 +61,24 @@ class AddFriendForm : public QObject
 
 public slots:
     void onUsernameSet(const QString& userName);
+    void onCurrentChanged(int index);
 
 private slots:
     void onSendTriggered();
     void onIdChanged(const QString &id);
     void onFriendRequestAccepted();
     void onFriendRequestRejected();
-    void onCurrentChanged(int index);
 
 private:
     void retranslateUi();
     void addFriendRequestWidget(const QString& friendAddress, const QString& message);
     void removeFriendRequestWidget(QWidget *friendWidget);
     void retranslateAcceptButton(QPushButton* acceptButton);
     void retranslateRejectButton(QPushButton* rejectButton);
+    void deleteFriendRequest(const QString &toxId);
+    void setIdFromClipboard();
```

Comment:
It seems, like first private section contain methods, second private -- property